salt/flink/init.sls

```diff
@@ -135,3 +138,12 @@ flink-copy_dependency_jar-{{ filename }}:
     - name: {{ flink_real_dir }}/lib/{{ filename }}
     - source: {{ file }}
 {% endfor %}
+
+{% for exec in execs %}
+flink_symlink_{{ exec }}:
+  file.symlink:
```

Comment:
In which situation is this symlink expected to be used (instead of the wrapped version prioritized in the user's PATH)?

Reply:
This Symlink will be used by yarn-wrapper.

Reply:
OK, no specific version needed then. Can the link point to the non-versioned 'flink_link_dir' (to simplify the upgrade process) instead?

Reply:
Done.